src/styles.scss

```diff
@@ -52,3 +53,7 @@ body {
     }
   }
 }
+.dark-backdrop {
+  background: #000;
+  opacity: 0.85 !important;
```

Comment:
Avoid the !important css rule as much as you can. Try using more specific css selector

Reply:
Fixed